temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid leaving debugging statements like `import pdb` in production code. It's better to remove it before committing.
```
import pdb
```